.eslintrc

```diff
@@ -16,6 +16,7 @@
     "camelcase": 0,
     "tree-shaking/no-side-effects-in-initialization": 0,
     "max-statements": 0,
-    "luma-gl-custom-rules/check-log-call": 1
+    "luma-gl-custom-rules/check-log-call": 1,
+    "spaced-comment": ["error", "always", {"exceptions": ["/"]}]
```

Comment:
Why would we diverge from this rule, all our code in all our repos follows it as does all other applications we work with? 

Making a local eslint-disable that is one thing, but changing it globally should really be something we agree on doing across our repos/in ocular-dev-tools.

Reply:
It's still on for almost all cases, but the '/' exception allows for this kind of header comment: https://github.com/uber/luma.gl/blob/master/examples/core/dof/app.js#L244

I can make that local to the file, though, until we discuss it further.

Reply:
Oh this style? 
```
///////////////////////////////////////
//
///////////////////////////////////////
```

That doesn't seem unreasonable, if you really want it widely used. I don't think you will get any major pushback on that, as long as the rule doesn't open any "other doors".

Worth remarking that our linter definitions are old, we should probably upgrade to a modern widely used standard. Uber has a new set of definitions that we should consider (we are using an older version).

We just need someone who cares enough about it to make a proposal.